rocky-raft/src/rocky/raft/dto/ConfigEntry.java

```diff
@@ -0,0 +1,40 @@
+package rocky.raft.dto;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class ConfigEntry implements Serializable {
+    private List<Address> currentConfig;
+    private List<Address> newConfig;
```

Comment:
Why do we need theses three variables? The fact that it's an old or new config doesn't have to be present in the log entry itself. A config log entry can be treated as a regular LogEntry with value = json of config and a special flag telling that it is a config entry. Server doesn't need to know separate addresses for old or new.